Gemfile

```diff
@@ -5,7 +5,7 @@ gem 'faraday'
 gem 'fast_jsonapi'
 gem 'pry'
 gem 'json'
-
+gem 'figaro', :git => 'https://github.com/bpaquet/figaro.git', :branch => 'sinatra'
 
```

Comment:
Nico said adding this to their figaro gem fixed their env var issue w/ Sinatra but no luck in our repo :( 
Made a separate ticket to address this.

Reply:
I plan on taking a look at this. I'll let you know if I can make any progress.